Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9-slim
+
+RUN apt-get update && apt-get install -y \
+    libpq-dev gcc && \
+    apt-get autoremove -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt /app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app/
+
+EXPOSE 8000
+
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
Consider using a separate entrypoint script for better maintainability and flexibility. This can help manage environment-specific configurations and commands more effectively.